docs/src/components/PropTable.js

```diff
@@ -171,10 +171,11 @@ export default function PropTable({
                     },
                     i
                   ) => {
+                    const hasSecondRow = description || responsive;
```

Comment:
Changes here fix an issue with the PropTable.

### Before

<img width="819" alt="Screen Shot 2020-11-30 at 4 00 42 PM" src="https://user-images.githubusercontent.com/127199/100681463-9ebfa400-3328-11eb-8988-dd38956c06a3.png">


### After
<img width="814" alt="Screen Shot 2020-11-30 at 4 00 56 PM" src="https://user-images.githubusercontent.com/127199/100681469-a1ba9480-3328-11eb-89ba-11b86fb197f8.png">